profile/nodejs.sh

```diff
@@ -24,10 +29,13 @@ detect_memory() {
 
 export PATH="$HOME/.heroku/node/bin:$HOME/bin:$HOME/node_modules/.bin:$PATH"
 export NODE_HOME="$HOME/.heroku/node"
-export NODE_ENV=${NODE_ENV:-production}
 
 calculate_concurrency
 
 export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
 export WEB_MEMORY=$WEB_MEMORY
 export WEB_CONCURRENCY=$WEB_CONCURRENCY
+
+if [ "$LOG_CONCURRENCY" = "true" ]; then
```

Comment:
Ah, here you go: https://github.com/realtymaps/heroku-buildpack-assets/blob/master/lib/environment.sh#L7

That's running `nodejs.sh` from the buildpack. The stuff in .profile.d is not supposed to be run during build - it creates the execution environment. You can get details here:

- https://devcenter.heroku.com/articles/buildpack-api